client/my-sites/plugins/plugin-site-jetpack/index.jsx

```diff
@@ -16,16 +16,32 @@ import PluginRemoveButton from 'my-sites/plugins/plugin-remove-button';
 import PluginSiteDisabledManage from 'my-sites/plugins/plugin-site-disabled-manage';
 import Site from 'blocks/site';
 
-export default React.createClass( {
+const PluginSiteJetpack = React.createClass( {
 
 	displayName: 'PluginSiteJetpack',
```

Comment:
If we're giving this a name, we have a babel transform to handle the displayName